plugins/base/src/main/kotlin/parsers/MarkdownParser.kt

```diff
@@ -42,10 +42,11 @@ open class MarkdownParser(
         when (tagName) {
             "see" -> {
                 val referencedName = content.substringBefore(' ')
+                val dri = externalDri(referencedName)
                 See(
                     parseStringToDocNode(content.substringAfter(' ')),
-                    referencedName,
-                    externalDri(referencedName)
+                    dri?.fqName() ?: referencedName,
```

Comment:
Shouldn't `referencedName` be used by default and the `fqname` as a fallback?

Reply:
I'd like to have a fq name every time it is possible because:

```
/**
* @throws RuntimeException sample runtime exception
* @throws java.lang.IllegalStateException another one
*/
void foo()
```
If we have such a case then it is nice to unify those names to display them accordingly. I'd like to have a fq name since it is most often unique and can be used to distinguish tags, but i am open for discussion.

Reply:
@sellophane ?